### PR TITLE
Feature/nginx codelitt site

### PR DESCRIPTION
**Pull request by antero10**
* Adding `nginx` codellit site `tag` with proper configurations for `redirect` the traffict for proper containers

### CODE REVIEW

**Review by codelitt — COMMENTED** [review]
@halissonvit @diogoribeiro please review

**Comment by codelitt** [comment]
LGTM

File: nginx/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM phusion/baseimage
```
Comment (halissonvit) [inline nginx/Dockerfile:1]: @antero10 I think that just inheriting from the phusion/baseimage is not enough... You still need to use their init file.

https://github.com/phusion/baseimage-docker#getting-started

**Review by diogoribeiro — COMMENTED** [review]
Looks good

File: nginx/codelitt-site/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+FROM codelittinc/nginx
```
Comment (halissonvit) [inline nginx/codelitt-site/Dockerfile:1]: I think we should not put files from private projects in a public repo

File: nginx/codelitt-site/sites/beta.codelitt.com.conf [hunk 3]
```diff
@@ -0,0 +1,40 @@
+server {
```
Comment (halissonvit) [inline nginx/codelitt-site/sites/beta.codelitt.com.conf:1]: I think we should not put files from private projects in a public repo

**Review by halissonvit — CHANGES_REQUESTED** [review]

File: nginx/codelitt-site/Dockerfile (hunk 2 above)
Comment (antero10) [reply nginx/codelitt-site/Dockerfile:1]: got it

**Pull request closed without merging**